bakerydemo/jinja2/tags/top_menu.html

```diff
@@ -1,14 +1,12 @@
-{% load navigation_tags wagtailcore_tags %}
-{% get_site_root as site_root %}
-
+{% set menuitems=top_menu(parent=get_site_root(), calling_page=page).menuitems %}
 {% for menuitem in menuitems %}
-    <li class="presentation {{ menuitem.title|lower|cut:" " }}{% if menuitem.active %} active{% endif %}{% if menuitem.show_dropdown %} has-submenu{% endif %}">
+    <li class="presentation {{ menuitem.title|lower|cut(" ") }}{{ "active" if menuitem.active }} {{ "has-submenu" if menuitem.show_dropdown }}">
```

Comment:
This "active" class does nothing currently so I’m not entirely sure why we have it.